backend/projects/migration_utils.py

```diff
@@ -0,0 +1,7 @@
+import re
+
+def name_to_slug(name: str, max_length: int = 64) -> str:
+    lowercase = name.lower()
+    no_whitespace = re.sub(r'\s', '_', lowercase)
+    cleaned = re.sub(r'[^a-z_]', '', no_whitespace)
+    return cleaned[:min(max_length, len(cleaned))]
```

Comment:
This function will yield empty strings if input consists of non-Latin characters only. But this function will only be used for a one-off migration, correct?

Reply:
Yes, it's only used here. In new models, the URI will be based on the `name`, which is a `SlugField` that can't be blank.